### PR TITLE
[hotfix][docs] Various doc enhancements around filesystem table connector and json

### PR DESCRIPTION
**Pull request by slinkydeveloper**
## What is the purpose of the change

Add more details to the documentation when using table api, filesystem connector and json.

## Brief change log

- Add additional sentence for minio users
- Specify that json format works for append only streams
- Specify that there is no defined ingestion order for the files in a directory when using the filesystem connector

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Review by Iqbalat87 — APPROVED** [review]

File: docs/content/docs/connectors/table/filesystem.md [hunk 1]
```diff
@@ -102,6 +102,12 @@ The file system connector supports multiple formats:
  - Canal-JSON: [canal-json]({{< ref "docs/connectors/table/formats/canal" >}}).
  - Raw: [raw]({{< ref "docs/connectors/table/formats/raw" >}}).
 
+## Source
+
+The file system connector can be used to read single files or entire directories into a single table.
+
+When using a directory as the source path, there is no defined order of ingestion for the files inside the directory.
```
Comment (matriv) [inline docs/content/docs/connectors/table/filesystem.md:109]: Maybe we should format it to make it stand out a bit more? Imho, looks like important info.

Comment (slinkydeveloper) [reply docs/content/docs/connectors/table/filesystem.md]: How about this?

Comment (matriv) [reply docs/content/docs/connectors/table/filesystem.md]: Looks, good! thank you!!

**Review by matriv — APPROVED** [review]
LGTM. Thank you!

**Pull request closed without merging**